lib/gui/app.js

```diff
@@ -219,3 +220,47 @@ app.controller('HeaderController', function(SelectionStateModel, OSOpenExternalS
   };
 
 });
+
+app.controller('ApplicationController', function($state,
+                                          SupportedFormatsModel,
+                                          OSDialogService,
+                                          AnalyticsService,
+                                          SelectionStateModel) {
+
+  this.formats = SupportedFormatsModel;
```

Comment:
Since we only use this model in one place (to join the support extensions), maybe worth exposing just the extensions?

```javascript
  /**
   * @summary Supported extensions
   * @constant
   * @type {String[]}
   * @public
   */
    this.supportedExtensions = SupportedFormatsModel.getAllExtensions();
```

And then simply join them before passing them to the directive?